Deprecation warnings from ipython breaking test collection

On my workstation (but oddly, not my laptop), I cannot seem to run tests any more as I get deprecation warnings from ipython using `async` and `await` internally. Not completely sure what to do about these - probably a newer version of ipython would resolve this, but on the other hand, astropy tests should not crash because of this.

```
_________________________________________________ ERROR collecting astropy/table/tests/test_jsviewer.py __________________________________________________
astropy/table/tests/test_jsviewer.py:11: in <module>
    import IPython  # pylint: disable=W0611
/usr/lib/python3/dist-packages/IPython/__init__.py:48: in <module>
    from .core.application import Application
/usr/lib/python3/dist-packages/IPython/core/application.py:25: in <module>
    from IPython.core import release, crashhandler
/usr/lib/python3/dist-packages/IPython/core/crashhandler.py:28: in <module>
    from IPython.core import ultratb
/usr/lib/python3/dist-packages/IPython/core/ultratb.py:124: in <module>
    from IPython.utils import path as util_path
/usr/lib/python3/dist-packages/IPython/utils/path.py:18: in <module>
    from IPython.utils.process import system
/usr/lib/python3/dist-packages/IPython/utils/process.py:19: in <module>
    from ._process_posix import system, getoutput, arg_split, check_pid
/usr/lib/python3/dist-packages/IPython/utils/_process_posix.py:24: in <module>
    import pexpect
/usr/lib/python3/dist-packages/pexpect/__init__.py:75: in <module>
    from .pty_spawn import spawn, spawnu
/usr/lib/python3/dist-packages/pexpect/pty_spawn.py:14: in <module>
    from .spawnbase import SpawnBase
E   DeprecationWarning: 'async' and 'await' will become reserved keywords in Python 3.7
```

--- a/astropy/tests/helper.py
+++ b/astropy/tests/helper.py
@@ -130,6 +130,7 @@ _modules_to_ignore_on_import = set([
     'scipy',
     'pygments',
     'ipykernel',
+    'IPython',   # deprecation warnings for async and await
     'setuptools'])
 _warnings_to_ignore_entire_module = set([])
 _warnings_to_ignore_by_pyver = {

